app/heron/factories/tube_rack.rb

```diff
@@ -41,7 +53,8 @@ def save
 
           Barcode.create!(asset: tube_rack, barcode: barcode, format: barcode_format)
 
-          @sample_tubes = create_tubes!(tube_rack)
+          create_recipients!
```

Comment:
'Recipients' is a bit of an odd word to use, although I can't think of anything better. Is it used elsewhere in our apps? I guess you could use 'Receptacles', but that is slightly inaccurate because Tubes are actually Labware, which contain a Receptacle.

Reply:
I wanted to represent the actual container from the content itself, container sound me as a bit ambiguous while recipient thought it gives the idea that it has contents. Didnt think about receptacle but seems fine as well.